### ISSUE TITLE
[Bug]: Prompt Templates are not working for Llama 2

### ISSUE THREAD

**Issue by PhucTranThanh** [BerriAI/litellm#401]
### What happened?

When I call the Llama 2 model on HuggingFace, it generates a text completion response instead of a chat response.

```python
import litellm

messages = [
    {
        "content": "I'm John, nice to meet you!",
        "role": "user"
    }
]

response = litellm.completion(model='huggingface/meta-llama/Llama-2-7b-chat-hf', messages=messages,
    api_key='',
    api_base='',
    max_tokens=2048
)

print(response['choices'][0]['message']['content'])
```

Here is the response:
```
I'm a software engineer and a tech enthusiast with a passion for building innovative solutions. I've worked on a variety of projects, from...
```

When I manually added the special tokens myself, it responded correctly.
```python
messages = [
    {
        "content": "[INST] I'm John, nice to meet you! [/INST]\n",
        "role": "user"
    },
]
```

Here is the response:
```
Hello John! *smiling* It's nice to meet you too! Is there something I can help you with or would you like to chat?
```

I'm expecting the special tokens should be automatically added according to the [docs](https://docs.litellm.ai/docs/providers/huggingface). I have not tested this on other models except Llama 2.

### Twitter / LinkedIn details

https://www.linkedin.com/in/phuc-thanh-tran

**Comment by ghost** [comment]
Thanks for the catch @PhucTranThanh  - i found the bug (it was a case-sensitive issue, which was causing the prompt to not get mapped). Fixed a push in `v0.1.699`

**Comment by ishaan-jaff** [comment]
the fix is on 0.1.700

**Comment by ghost** [comment]
closing - @PhucTranThanh please reopen if the issue persists.
